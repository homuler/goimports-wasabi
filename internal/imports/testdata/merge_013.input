// merge empty import declarations
package main

import ()
import _ "fmt"
