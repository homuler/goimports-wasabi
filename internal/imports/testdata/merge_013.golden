// merge empty import declarations
package main

import (
	_ "fmt"
)
